src/adaptermanager.js

```diff
@@ -55,24 +55,41 @@ function getBids({bidderCode, requestId, bidderRequestId, adUnits}) {
           sizes: sizes,
           bidId: bid.bid_id || utils.getUniqueIdentifierStr(),
           bidderRequestId,
-          requestId
+          auctionId
         });
       }
       );
   }).reduce(flatten, []).filter(val => val !== '');
 }
 
-exports.callBids = ({adUnits, cbTimeout}) => {
-  const requestId = utils.generateUUID();
-  const auctionStart = Date.now();
+function getAdUnitCopyForPrebidServer(adUnits) {
+  let adaptersServerSide = _s2sConfig.bidders;
+  let adUnitsCopy = utils.cloneJson(adUnits);
 
-  const auctionInit = {
-    timestamp: auctionStart,
-    requestId,
-    timeout: cbTimeout
-  };
-  events.emit(CONSTANTS.EVENTS.AUCTION_INIT, auctionInit);
```

Comment:
Now emitting this event from different place. Moved to auction.js. Still checking for analytics adapters for support.